drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/pom.xml

```diff
@@ -157,9 +157,27 @@
       <artifactId>kie-dmn-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.kie</groupId>
+      <artifactId>kie-dmn-core</artifactId>
+      <classifier>tests</classifier>
```

Comment:
Why are you adding this as classifier? This project already declare `kie-dmn-core` as test dependency

Reply:
@danielezonca 
Because the class used is declared inside the "test" directory, so it is not inherited by the "normal" dependency